source/computer/commands.rst

```diff
@@ -1,2 +1,100 @@
 常用命令
 ========
+
+Linux 下有成百上千个命令，每个命令都有众多选项。这一节介绍一些常用命令的常用用法。
+
+
+.. rubric:: 命令目录
+
+.. hlist::
+    :columns: 4
+
+    -   压缩与解压
+
+        - `tar`_
+
+    -   文件查看
+
+        - `cat`_
+        - `head`_
+        - `tail`_
+
+    -   文件处理
+
+        - `gawk`_
+        - `cut`_
```

Comment:
`cut` and `paste` are useful for formal data processing, but I think we're recommending performing data processing using Python, so `cut` and `paste` are less useful, right?

Reply:
Of course, we shoule use python. but do we want new babies kown them and use them if they just need to do some tests in CML?

Reply:
But why do they need to run or test `cut` in CLI?

Reply:
you mean we may use gawk instead?

Reply:
No, I mean we should use pure Python codes for data processing, no calling of `cut`, `paste`, `gawk` etc.

Reply:
I agree, but we may use them sometimes in teminal.

Reply:
I may use it once a year 😃

Reply:
> I may use it once a year 😃

me, too. please remove it or them.😅😅